nextcloud_update.sh

```diff
@@ -261,11 +261,22 @@ then
     yes no | pecl upgrade redis
     systemctl restart redis-server.service
 fi
-
-# Double check if redis.so is enabled
-if ! grep -qFx extension=redis.so "$PHP_INI"
+# Remove old redis
+if grep -qFx extension=redis.so "$PHP_INI"
+then
+    sed -i "s|extension=redis.so||g" "$PHP_INI"
```

Comment:
Use delete here instead. This works but leaves a line in the file.

`> sed '/redis.so/d' file`

Reply:
done